schema/me/update_conversation.js

```diff
@@ -0,0 +1,32 @@
+import impulse from '../../lib/loaders/impulse';
+import gravity from '../../lib/loaders/gravity';
+import { ConversationType } from './conversations';
+import {
+  GraphQLString,
+  GraphQLNonNull,
+} from 'graphql';
+const { IMPULSE_APPLICATION_ID } = process.env;
+
+export default {
+  type: ConversationType,
+  decription: 'Updating buyer outcome of a conversation.',
+  args: {
+    buyer_outcome: {
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    id: {
+      type: new GraphQLNonNull(GraphQLString),
+    },
+  },
+  resolve: (root, { buyer_outcome, id }, request, { rootValue: { accessToken } }) => {
+    if (!accessToken) return null;
+    return gravity.with(accessToken, { method: 'POST' })('me/token', {
```

Comment:
We should definitely also create loaders for these that are pre-authenticated before passing them into the `rootValue` object.